Libraries/xcsdk/Headers/xcsdk/SDK/Manager.h

```diff
@@ -88,7 +89,7 @@ class Manager {
     /*
      * Load from a developer root. Returns nullptr on error.
      */
-    static std::shared_ptr<Manager> Open(libutil::Filesystem const *filesystem, std::string const &path);
+    static std::shared_ptr<Manager> Open(config::Config const *config, libutil::Filesystem const *filesystem, std::string const &path);
```

Comment:
Yeah, there's definitely different standards. I personally like [Google's reasoning](https://google.github.io/styleguide/cppguide.html#Reference_Arguments) for avoiding non-const references: "References can be confusing, as they have value syntax but pointer semantics."